price-feeder/oracle/oracle.go

```diff
@@ -98,12 +99,19 @@ func (o *Oracle) Start(ctx context.Context) error {
 
 		default:
 			o.logger.Debug().Msg("starting oracle tick")
+
+			startTime := time.Now()
+
 			if err := o.tick(); err != nil {
+				telemetry.IncrCounter(1, "failure", "tick")
 				o.logger.Err(err).Msg("oracle tick failed")
 			}
 
 			o.lastPriceSyncTS = time.Now()
 
+			telemetry.MeasureSince(startTime, "runtime", "tick")
```

Comment:
I don't think we should include `time.Sleep` as a part of this metric, although it's technically a part of the `tick`, would make the data more difficult to evaluate